hathor/consensus/consensus_settings.py

```diff
@@ -112,5 +130,12 @@ def _get_valid_vertex_versions(self, include_genesis: bool) -> set[TxVersion]:
 
         return versions
 
+    @override
+    def get_peer_hello_hash(self) -> str | None:
+        data = b''
+        for signer in self.signers:
```

Comment:
This code enforces the order of the signers and I think this is a good verification. I think you can safely remove the `sorted()` from the verification so we ensure that the signed id is the same for all peers and that we can check the signer id simply by checking the settings file.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/1078/commits/b58b125dae7da71d39f750fa53d8729f4efe3463